cloudify_cli/utils.py

```diff
@@ -385,16 +394,25 @@ def get_rest_port():
     return cosmo_wd_settings.get_rest_port()
 
 
-def get_protocol():
+def get_rest_protocol():
+    cosmo_wd_settings = load_cloudify_working_dir_settings()
+    return cosmo_wd_settings.get_rest_protocol()
+
+
+def get_rest_host():
+    return get_management_server_ip()
+
+
+def get_rest_server_public_certificate():
     cosmo_wd_settings = load_cloudify_working_dir_settings()
-    return cosmo_wd_settings.get_protocol()
+    return cosmo_wd_settings.get_rest_server_public_certificate()
 
 
 def get_management_server_ip():
```

Comment:
perhaps this function should be renamed to get_rest_host
and have
`get_management_server_ip = get_rest_host` instead of the previous suggestion